xsstrike.py

```diff
@@ -17,7 +17,10 @@
     except ImportError:
         import os
         print ('%s fuzzywuzzy isn\'t installed, installing now.' % info)
-        os.system('pip3 install fuzzywuzzy')
+        ret_code = os.system('pip3 install fuzzywuzzy')
```

Comment:
Have we documented this behavior? On systems I use, this old school system call might install the package somewhere else ... so the install would return 0 but import would still fail.